src/pages/index.js

```diff
@@ -167,17 +268,36 @@ function getCardElement(data) {
     openModal(previewModal);
   });
 
+  function handleDeleteCard(cardElement, data) {
+    selectedCard = cardElement;
+    selectedCardId = data._id;
+
+    openModal(deleteModal);
+  }
+
+  function handleDeleteSubmit(evt) {
+    setButtonText(evt.submitter, true, "Delete", "Deleting...");
+    api
+      .deleteCard(selectedCardId)
+      .then(() => {
+        selectedCard.remove();
+        closeModal(deleteModal);
+      })
+      .catch(console.error);
+  }
+
   cardElementTitle.textContent = data.name;
   cardElementImage.src = data.link;
   cardElementImage.alt = data.name;
 
+  deleteModalForm.addEventListener("submit", (evt) => {
+    evt.preventDefault();
+    handleDeleteSubmit(evt);
```

Comment:
* When I delete cards, there are errors https://snipboard.io/D2oqCB.jpg

Please, remove it from the `getCardElement` function because the `submit` handler should be added only **once** outiside the creation function


Please, move it onto line 310